src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +29,37 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => {
+                cancelSelectedTodo();
+              }}
```

Comment:
```suggestion
              onClick={cancelSelectedTodo}
```
since your function doesn't accept any arguments you can set it as `onClick` function directly